hyperstart_agent.go

```diff
@@ -416,6 +416,13 @@ func (h *hyper) startOneContainer(pod Pod, c Container) error {
 		Process: process,
 	}
 
+	if c.config.Resources.VCPUs != 0 {
+		container.Constraints = hyperstart.Constraints{
+			CPUQuota:  c.config.Resources.CPUQuota,
+			CPUPeriod: c.config.Resources.CPUPeriod,
+		}
+	}
+
```

Comment:
Isn't it great ! Nothing needs to be done for `kata_agent.go` since this is already into the spec :)